articles/post.php

```diff
@@ -2,6 +2,13 @@
 $post_share_url = esc_url( get_permalink() );
 $post_share_title = rawurlencode( spine_get_title() );
 $post_share_placement = spine_get_option( 'post_social_placement' );
+$spine_social_options = spine_social_options();
+if ( ! empty( $spine_social_options['twitter'] ) ) {
+	$twitter_array = array_filter( explode( '/', $spine_social_options['twitter'] ) );
+	$twitter_handle = array_pop( $twitter_array );
```

Comment:
Wrapping `$twitter_handle` in `esc_attr()` would be good here so that we're sure it's safe when output as part of the URL later in the page. Overcautious in some ways, but the data is coming from the database.